take-your-first-steps-with-rust/get-started-with-rust/includes/3-hello-world.md

```diff
@@ -1 +1,57 @@
 # Exercise: Hello World
+
+Now that you’ve installed Rust, let’s write your first Rust program that prints the message "Helo,
+world!" to the console.
+
+
+## Creating a new directory to organize our code
+
+We'll start by creating a directory for storing all the code in this learning path
+(`rust-learning-path`) and then creating a new sub-directory to keep the source code for this
+exercise.
+
+For Linux, macOS, and PowerShell on Windows, enter this:
+
+    $ mkdir ~/rust-learning-path
+    $ cd ~/rust-learning-path
+    $ mkdir hello_world
+    $ cd hello_world
+
+For Windows CMD, enter this:
+
+    > mkdir "%USERPROFILE%\rust-learning-path"
+    > cd /d "%USERPROFILE%\rust-learning-path"
+    > mkdir hello_world
+    > cd hello_world
+
+
+## Writing our first Rust code
+
+Next, create a new file named `main.rs` and use your editor to write the following code into it:
```

Comment:
We should specify exactly where we want learners to run this code and not make assumptions they have any code editor. And if we specify an editor like VS Code, then we should provide instructions on how to install that editor first. 

Alternatively, we could just use the playground, but if we do that, I wonder would it be necessary to install Rust in the previous unit then.

Let me know what you think here.

Reply:
I was planning to be agnostic about the student's text editor.

Do you think it would be possible to mention in the Prerequisites section that students should be familiar with a text editor of their choice?

I ask this because we could refer the user to [this link](https://www.rust-lang.org/tools), which is on Rust's official page and contains detailed instructions for using the most common editors with the Rust programming language.

If this is not the case, I can include a previous unit containing the VS Code installation instructions, no problem.

Reply:
@tilacog from all the other Learn content I've seen, they either provide some kind of editor experience in the platform or recommend VS Code. To be in line with that, I'm thinking it would be better to add a previous unit with the instructions to download that. Thank you!